src/runtime/cli/main.go

```diff
@@ -451,7 +453,7 @@ func userWantsUsage(context *cli.Context) bool {
 func fatal(err error) {
 	kataLog.Error(err)
 	fmt.Fprintln(defaultErrorFile, err)
-	exit(1)
+	exitFunc(1)
```

Comment:
Same comment as above.

Reply:
fatal() will be used in TestMainFatal(), if remove exitFunc() in main.go, unit test will be failed. Because  exitFunc() can't be os.Exit() when in unit test

```go
	var exitStatus int
	savedExitFunc := exitFunc

	exitFunc = func(status int) { exitStatus = status }

	savedErrorFile := defaultErrorFile
```